Dockerfile

```diff
@@ -1,9 +1,18 @@
-FROM node:12.0.0
-
-WORKDIR /usr/src/app
-
+FROM node:12.18.2-alpine AS build
```

Comment:
Using a so-called multi-stage build here. This stage uses `devDependencies` to build the app, while the stage below (line 8 ff.) will be smaller because it only has JS, not TS files and dependencies.